OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -134,26 +133,32 @@ public override bool Tick(Actor self)
 			var desiredFacing = delta.HorizontalLengthSquared != 0 ? delta.Yaw.Facing : aircraft.Facing;
 			var isAirborne = self.World.Map.DistanceAboveTerrain(pos).Length >= aircraft.Info.MinAirborneAltitude;
 
-			if (!isAirborne)
-				QueueChild(new TakeOff(self));
+			QueueChild(new TakeOff(self));
```

Comment:
This makes helicopters play their TakeOff sound every tick while attacking!

Reply:
Fixed.